frontend/src/sass/includes/rc-collapse.scss

```diff
@@ -0,0 +1,61 @@
+.rc-collapse {
+  background-color: #f7f7f7;
+  border-radius: 3px;
+  border: 1px solid #d9d9d9;
+}
+.rc-collapse-anim-active {
+  transition: height 0.2s ease-out;
+}
+.rc-collapse > .rc-collapse-item {

```

Comment:
If this is a library, can we not import it as a react/gulp module? Seems wrong to track it in git